js/layer_manager_row.ts

```diff
@@ -0,0 +1,246 @@
+import type { AnyModel, RenderProps } from "@anywidget/types";
+import { html, css, LitElement, TemplateResult, nothing } from "lit";
+import { property } from "lit/decorators.js";
+import { classMap } from 'lit/directives/class-map.js';
+
+import { legacyStyles } from './ipywidgets_styles';
+import { materialStyles } from "./material_styles";
+import { loadFonts } from "./utils";
+
+interface LayerManagerRowModel {
```

Comment:
One more thing! Can you `export` this interface, I need it for tests! However, I'm happy to do it in my PR if you prefer to localize changes like that.